modules/bankfeed/api/bankfeed.ts

```diff
@@ -0,0 +1,18 @@
+import { Request, Response } from 'express';
+import * as core from 'express-serve-static-core';
+import BankData from '../data/bandata';
+
+module.exports = function (app: core.Express) {
+  // Get a list of bank statement
+  app.get('/', (req: Request, res: Response) => {
```

Comment:
please can this be '/bank/feed/:qty'

qty being the amount of transactions they would like in their statement array. We already have a util to get the qty from the request. getQtyFromRequest